bots/delete_access_key.py

```diff
@@ -0,0 +1,46 @@
+"""
+## delete_access_key
+What it does: Delete an IAM user access key
+Usage : AUTO: delete_access_key
+Limitations: none
+
+This bot is relevant to logic only and it's Running Following the CreateAccessKey event
+"""
+
+import boto3
+from botocore.exceptions import ClientError
+import bots_utils
+
+EVENT_NAME = 'CreateAccessKey'
+DEFAULT_CLOUDTRAIL_LOOKUP_TIME_DIFF=5
+
+
+def run_action(boto_session, rule, entity, params):
+    # create iam resource
+    iam_resource = boto3.resource('iam')
```

Comment:
please use boto3.client

Reply:
the method i'm using is only from an boto3.resource